explorer/src/components/SolanabeachComponents/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, Fragment } from "react";
+import io from "socket.io-client";
+import NetworkStats from "./NetworkStats";
+import { Cluster, useCluster } from "../../providers/cluster";
+
+class LandingPageInner extends Component {
+  constructor(props) {
+    super(props);
+
+    // Connect socket before component is mounted
+
+    this.socket = io("https://api.solanabeach.io:8443/mainnet");
```

Comment:
We already have a socket connection in `providers/stats/solanaBeach`. It'd be nice to stick with just one. Could you try to integrate all of the views with that data provider? I already set it up to cache the current content while navigating around the explorer so that there isn't a lag every time you return to the stats page